packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.styling.ts

```diff
@@ -44,5 +44,16 @@ export const stylingSettings: UseStylingOptions<RadioGroupProps, RadioGroupSlotP
       }),
       ['requiredColor', 'requiredPadding'],
     ),
+    options: buildProps(
```

Comment:
I was thinking the name 'layout' might make more sense here (the style props below are all grouped together as 'layout' in [react native docs](https://reactnative.dev/docs/flexbox)), and maybe the 'layout' prop you added could be 'labelPosition' or 'labelDirection' or something

Reply:
As a slot I think options or optionsContainer makes sense, since it tells us where in the UI the slot is being used. I think layout is pretty vague and clients can always add non-layout props to any slot, so we shouldn't rely on the props to decide what to call a slot.

Reply:
Leaving this named as options for now. Please let me know if I should change.